GiEnJul/ClientApp/src/components/admin/AdminPage.tsx

```diff
@@ -1,23 +1,80 @@
-import { withAuthenticationRequired } from "@auth0/auth0-react";
+import { useAuth0, withAuthenticationRequired } from "@auth0/auth0-react";
 import { Container } from "@material-ui/core";
 import * as React from "react";
+import { useEffect, useState } from "react";
 import LoadingPage from "../../common/components/LoadingPage";
+import InstitutionMacro from "../institution/InstitutionMacro";
 import LogOutButton from "../login/LogOutButton";
 import AdminMenu from "./common/AdminMenu";
 import Completed from "./connections/Completed";
 import Suggested from "./connections/Suggested";
 
 function AdminPage() {
-  return (
-    <>
-      <LogOutButton></LogOutButton>
-      <Container maxWidth="xl">
-        <AdminMenu />
-        <Suggested />
-        <Completed />
-      </Container>
-    </>
-  );
+  const { user, getAccessTokenSilently } = useAuth0();
+  const [appMetadata, setappMetadata] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    const getappMetadata = async () => {
+      const domain = "dev-r7fmessb.eu.auth0.com";
```

Comment:
this could be stored somewhere to be globally available

Reply:
yea, I already have, hehe. Just forgot to use it here.
Have it stored in a .env file